apptools/naming/tests/test_pyfs_context.py

```diff
@@ -293,15 +275,11 @@ def test_destroy_subcontext(self):
         # Try to destroy it.
         self.assertRaises(NotContextError, context.destroy_subcontext, "sub/a")
 
-        return
```

Comment:
This is an early termination of the test, which means the subsequent `assertRaises` has not been run in the past, and is now being run. Luckily, it still passes.

Reply:
yeah. i was looking out for this when i first pushed the changes.